cms/templates/widgets/header.html

```diff
@@ -43,7 +43,7 @@ <h2 class="info-course">
         </a>
       </h2>
 
-      <nav class="nav-course nav-dd ui-left">
+      <nav class="nav-course nav-dd ui-left" aria-label="${_('{course_name}').format(course_name=context_course.display_name_with_default)}">
```

Comment:
@ahsan-ul-haq since the current course is not important for understanding which nav this is (what's important is that its the Course nav as opposed to the Site nav or the Course Tools nav, etc.) I recommend changing the aria-label to the more generic "Course".  It should also be translatable.